app/controllers/api/v1/graphql_controller.rb

```diff
@@ -53,7 +53,7 @@ def parse_graphql_result
         # Mutations are not batched, so we can return errors in the root
         rescue ActiveRecord::RecordInvalid, RuntimeError, NameError, GraphQL::Batch::NestedError => e
           @output = parse_json_exception(e)
-          CheckSentry.notify(e)
+          CheckSentry.notify(e) unless e.is_a?(ActiveRecord::RecordInvalid)
```

Comment:
I think rather than doing a clause here, we should use the Sentry ruby gem's configuration for ignoring certain exceptions. Here are the docs on how to use: https://docs.sentry.io/platforms/ruby/configuration/options/, cmd+f `excluded_exceptions`.

This would go in the Sentry config (check-api/config/initializers/02_sentry.rb)